src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #eef;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
I don't think you should explicitly set width here, let the content inside dictate its parent width